### PR TITLE
Remove usePureJavaScript option and checks.

### PR DESCRIPTION
**Pull request by davidlehn**

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -100,9 +92,8 @@ api.canonize = util.callbackify(async function(dataset, options) {
  * Synchronously canonizes an RDF dataset.
  *
  * @param dataset the dataset to canonize.
- * @param [options] the options to use:
- *          [algorithm] the canonicalization algorithm to use, `URDNA2015` or
- *            `URGNA2012` (default: `URGNA2012`).
+ * @param options the options to use:
+ *          algorithm the canonicalization algorithm to use: `URDNA2015`.
```
Comment (mattcollier) [inline lib/index.js:98]: `algorithm` is required now, and there is no default?

**Review by mattcollier — APPROVED** [review]
Just one question about documentation.

**Review by dlongley — APPROVED** [review]

File: lib/index.js (hunk 1 above)
Comment (dlongley) [reply lib/index.js:98]: I think the docs were just wrong before.

Comment (davidlehn) [reply lib/index.js:98]: Yeah, I was just making the code match the docs.  Same in rdf-canonize.

**Pull request merged**